### PR TITLE
Add Light Unit Slider for Auto Exposure Limits

### PR DESCRIPTION
**Pull request by johnpars**
### Purpose of this PR
Title mostly summarizes the content of this PR; a light unit slider has been added for the limit values for Exposure volume **Automatic** and **Automatic Histogram** modes. Additionally, the default limit values have been adjusted to **-1** and **16** (changed from **-10** and **20**) after discussion with lighting artist.

AutomaticExposure

Additionally, the PR includes a small bug fix that corrects the caution tooltip value warning. If you used an exposure value below the recommended amount, it would tell you the incorrect minimum value.

---
### Testing status
- Manually tested undo & redo.
- Lightly tested exposure etc. Since this UI addition is simply a view to the data, no issues are expected here.

---
### Comments to reviewers
~

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Components/Exposure.cs [hunk 1]
```diff
@@ -49,14 +49,14 @@ public sealed class Exposure : VolumeComponent, IPostProcessComponent
         /// This parameter is only used when <see cref="ExposureMode.Automatic"/> or <see cref="ExposureMode.CurveMapping"/> is set.
         /// </summary>
         [Tooltip("Sets the minimum value that the Scene exposure can be set to.")]
-        public FloatParameter limitMin = new FloatParameter(-10f);
+        public FloatParameter limitMin = new FloatParameter(-1f);
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Components/Exposure.cs:52]: Note be sure to launch graphics test :)

**Review by sebastienlagarde — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Components/Exposure.cs (hunk 1 above)
Comment (johnpars) [reply com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Components/Exposure.cs:52]: Launched!

**Review by TomasKiniulis — APPROVED** [review]
Both Automatic and Automatic  Histogram exposure sliders work correctly.

- Switching through icon button correctly changes value to a set section of the slider
- Slider and button undo/redo works
- Warning icon shows up with invalid values
- Going back from other exposure modes keep the correct set values

File: com.unity.render-pipelines.high-definition/Editor/PostProcessing/ExposureEditor.cs [hunk 2]
```diff
@@ -204,18 +204,18 @@ public override void OnInspectorGUI()
             }
         }
 
-        // TODO: See if it's possible to refactor into a custom VolumeParameterDrawer
-        void DoFixedExposureField(SerializedDataParameter fixedExposure)
+        // TODO: See if this can be refactored into a custom VolumeParameterDrawer
```
Comment (RSlysz) [inline com.unity.render-pipelines.high-definition/Editor/PostProcessing/ExposureEditor.cs:207]: Could be good to actually do it. This should be doable AFAIK. It'll let user the possibility to use this in their volume inspectors. Here it is not usable to users.

**Review by RSlysz — APPROVED** [review]
Seams good. Though we could profit to actually do the TODO

**Pull request merged**